doc/classes/TileMap.xml

```diff
@@ -158,18 +158,21 @@
 			</argument>
 			<argument index="6" name="autotile_coord" type="Vector2" default="Vector2( 0, 0 )">
 			</argument>
+			<argument index="7" name="atlas_priority" type="bool" default="true">
+			</argument>
 			<description>
 				Sets the tile index for the cell given by a Vector2.
 				An index of [code]-1[/code] clears the cell.
 				Optionally, the tile can also be flipped, transposed, or given autotile coordinates. The autotile coordinate refers to the column and row of the subtile.
 				[b]Note:[/b] Data such as navigation polygons and collision shapes are not immediately updated for performance reasons.
+				[b]Note:[/b] [code]autotile_coord[/code] is also used to define the column and row of [enum TileSet.TileMode.ATLAS_TILE], for this you must pass [code]false[/code] to [code]atlas_priority[/code] to disable priority.
```

Comment:
To reference a specific constant, you should use `[constant TileSet.ATLAS_TILE]`.

Reply:
Thanks.